packages/test/test-end-to-end-tests/src/test/SummarizeFetchValidation.spec.ts

```diff
@@ -29,12 +29,10 @@ import {
 	waitForContainerConnection,
 } from "@fluidframework/test-utils/internal";
 
-// Note GC needs to be disabled.
```

Comment:
Not true anymore?  Or I guess disabling summary does the trick...

Reply:
I believe this is just a copy-paste. GC doesn't need to be disabled here.